internal/daemon/controller/handlers/groups/group_service.go

```diff
@@ -710,7 +710,7 @@ func validateAddGroupMembersRequest(req *pbs.AddGroupMembersRequest) error {
 			badFields["member_ids"] = fmt.Sprintf("Must only contain valid user ids but found %q.", id)
 			break
 		}
-		if id == "u_recovery" {
+		if id == perms.RecoveryUserId {
```

Comment:
This should really be in the `globals` package as there are other places that use `u_recovery`. I'd hold off switching things for a separate PR though as it will add a lot of extra files/changes to this PR.

Reply:
I'll do this in a follow-up PR